ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/MSVS_2019/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+################################
+# Visual Studio Community 2019 #
+################################
+
+- name: Test if VS 2019 is installed
+  win_stat:
+    path: 'C:\Program Files (x86)\Microsoft Visual Studio\2019'
+  register: vs2019_installed
+  tags: MSVS_2019
+
+- name: Download Visual Studio Community 2019
+  win_get_url:
+    url: 'https://aka.ms/vs/16/release/vs_community.exe'
+    dest: 'C:\temp\vs_community19.exe'
+    force: no
+    checksum: 771a4c18e31ccc341097af13302792331817ae81fce20f8c99799163d87733d4
```

Comment:
Checksum has changed ( https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/831/OS=Win2012,label=vagrant/console )
Confirmed it to be 
`03f1768299410a2c359f61c88abbe038c5bcfd51d71854cf6437e633ebdefa06`

Reply:
Checksum updated